RELEASE_NOTES.md

```diff
@@ -1,5 +1,16 @@
 ﻿RELEASE NOTES
 
+**Version**: 1.1
+
+This versions requires:
+* Shotgun for Unity 1.1 (com.unity.integrations.shotgun)
+
+NEW FEATURES
+* Direct Unity project launch(no need for Unity Hub)
+* Now working in-process
+* One-click install for Mac/Win
```

Comment:
I don't think those 2 needs to be mentionned here; that's more com.unity.integrations.shotgun functionality.